src/logic/operate.js

```diff
@@ -8,15 +8,14 @@ const operate = (numberOne, numberTwo, operation) => {
     case '-':
       return nOne.minus(nTwo).toString();
     case '+':
-      return (nOne + nTwo);
+      return (nOne.plus(nTwo)).toString();
     case 'X':
       return nOne.times(nTwo).toString();
-    case '÷':
+    case '/':
       if (nTwo === '0') {
-        return '0';
+        return Infinity;
```

Comment:
This doesn't handle the division by zero error, I still get an error when I try dividing by zero